crypto/rand/rand_unix.c

```diff
@@ -448,14 +441,14 @@ void rand_pool_keep_random_devices_open(int keep)
  */
 size_t rand_pool_acquire_entropy(RAND_POOL *pool)
 {
-#  if defined(OPENSSL_RAND_SEED_NONE)
+#  ifdef OPENSSL_RAND_SEED_NONE
```

Comment:
please no polishing in this PR

Reply:
I only recently cleaned up the mixture of `#ifdef`'s and `#if defined(...)` to use the latter consistently (see d90e128be6e).

Reply:
Note: The latter style has advantages, even if there is only one condition to check: If you need to add another condition, you don't need to change from `#ifdef` to `#if defined`.